ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -73,6 +78,45 @@ export default class StatusDropdown extends React.Component {
     };
   };
 
+  getNumberOfNonWorkingDays(createdAt, dueDate) {
+    let count = 0;
+    const curDate = new Date(createdAt.getTime());
+    const sunday = 0;
+    const saturday = 6;
+
+    curDate.setHours(0, 0, 0, 0);
+    dueDate.setHours(0, 0, 0, 0);
+
+    if (curDate.getDay() === sunday) {
+      return 1;
+    }
+
+    if (curDate.getDay() === saturday) {
+      return 2;
+    }
+
+    if (dueDate.getDay() === saturday || dueDate.getDay === sunday) {
```

Comment:
Here `dueDate.getDay === sunday` should be replaced to `dueDate.getDay() === sunday`.